lib/sql-string.js

```diff
@@ -1,28 +1,10 @@
 'use strict';
```

Comment:
I've cleaned this file up a bit to only include the parts we need. I looked into replacing in with https://www.npmjs.com/package/sqlstring, but that's not going to happen - we have a lot of modifications, and that module only supports mysql